extensions-core/mysql-metadata-storage/src/main/java/io/druid/metadata/storage/mysql/MySQLConnector.java

```diff
@@ -163,17 +162,19 @@ protected int getStreamingFetchSize()
   @Override
   public boolean tableExists(Handle handle, String tableName)
   {
-    // ensure database defaults to utf8, otherwise bail
-    boolean isUtf8 = handle
-        .createQuery("SELECT @@character_set_database = 'utf8'")
-        .map(BooleanMapper.FIRST)
+    String databaseCharset = handle
+        .createQuery("SELECT @@character_set_database")
+        .map(StringMapper.FIRST)
         .first();
 
-    if (!isUtf8) {
+    if (!databaseCharset.matches("utf.*")) {
       throw new ISE(
-          "Database default character set is not UTF-8." + System.lineSeparator()
-          + "  Druid requires its MySQL database to be created using UTF-8 as default character set."
+          "Druid requires its MySQL database to be created with an UTF charset, found `%1$s`.%n" +
```

Comment:
Can you remove newlines ? It confuses lots of logging systems.

Reply:
Sure.